var/spack/repos/builtin/packages/bdw-gc/package.py

```diff
@@ -0,0 +1,52 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+
+
+class BdwGc(Package):
+    """The Boehm-Demers-Weiser conservative garbage collector is a garbage
+    collecting replacement for C malloc or C++ new."""
+
+    homepage = "http://www.hboehm.info/gc/"
+    url      = "http://www.hboehm.info/gc/gc_source/gc-7.4.4.tar.gz"
```

Comment:
So Spack auto-detected the package name for this one as gc, but I manually changed it to bdw-gc for three reasons:

1. The [GitHub repo](https://github.com/ivmai/bdwgc) is named bdwgc
2. The [Homebrew package](https://github.com/Homebrew/homebrew-core/blob/master/Formula/bdw-gc.rb) is named bdw-gc
3. There may be other garbage collectors added to Spack someday?

Let me know if you have any strong opinions on this. I really don't care either way.